tests/fsharpqa/Source/Conformance/InferenceProcedures/TypeInference/IgnoreUnitParameters.fs

```diff
@@ -0,0 +1,46 @@
+// #Regression #TypeInference 
+// Regression test for https://github.com/Microsoft/visualfsharp/issues/1749
+// Type Inference
+namespace N
+module InferUnit = 
+
+    let run1 (f: unit -> string) = f()
+    
+    let a = run1 (fun () -> "hello world")
+    let a' = run1 (fun _ -> "hello world")
+
+    
+    if a <> a' then
+        failwith "bug in a"
+    
+
+    
+    let run2 (f:System.Func<unit,string>) = f.Invoke()
+
+    let b = run2 (System.Func<unit,string>(fun () -> "hello world 2"))
+    let b' = run2 (System.Func<unit,string>(fun _ -> "hello world 2"))
```

Comment:
This is a scenario too.

let b'' = run2 (System.Func<unit,string>(fun x -> "hello world 2"))